src/mutation_data.cpp

```diff
@@ -119,6 +114,11 @@ const std::map<std::string, mutation_category_trait> &mutation_category_trait::g
     return mutation_category_traits;
 }
 
+const mutation_category_trait &mutation_category_trait::get_category( std::string category_id )
+{
+    return mutation_category_traits[category_id];
```

Comment:
When `get`ting, it's better to use `map::find` instead of `map::operator[]` - the latter is non-`const` and will create an entry if it's missing.